src/gval/homogenize/spatial_alignment.py

```diff
@@ -139,6 +138,74 @@ def _rasters_intersect(
         return rasters_intersect_bool
 
 
+def _check_dask_array(original_map: Union[xr.DataArray, xr.Dataset]) -> bool:
```

Comment:
Maybe put this function somewhere else under homogenize? It's not really about spatial alignment, it's more about array backends.

Reply:
I moved this method to numerical alignment